hnginternship5/ziki
# Restore login modal and logo fix

#### What does this PR do?
* It restores the login modal for the sidebar.
* It fixes the logo alignment problem.
* It gives every page a unique title in `header.html` and adds a favicon too.
#### Description of Task to be completed?
* Fix login modal in `sidebar.html`.
* Create modal for post deletion
* add dropdown for options on drafts page 
#### How should this be manually tested?
* Open the site on your browser
* Click the login button on the sidebar
* You should see the login dialog
#### Any background context you want to provide?
* post delete modal has been started in this PR but not yet completed
* option dropdown on drafts page is still not right.
#### What are the relevant pivotal tracker stories?
[#165478056](https://www.pivotaltracker.com/story/show/165478056)
#### Screenshots (if appropriate)
![tabletscreen](https://i.postimg.cc/503rfgzD/localhost-8000-editor.png)
* tablet screen

![pc screen](https://i.postimg.cc/rwB3jKcZ/localhost-8000-editor-1.png)
* pc screen

![mobile screen](https://i.postimg.cc/TwGS8FKr/localhost-8000-editor-3.png)
* mobile screen
#### Questions/Comments:
* none
